src/installer/prepare-artifacts.proj

```diff
@@ -118,11 +118,6 @@
         <PublishFlatContainer>true</PublishFlatContainer>
       </ItemsToPush>
 
-      <ItemsToPush Include="@(WorkloadsVSInsertionFile)">
```

Comment:
After this is removed, where do the assets from the swix projects get picked up for publishing?

Reply:
😢 good point. I should probably add the new zip files to the item group

Reply:
I think I fixed this. The zips are unique named, so we can just put them in the same folder